mlflow/sklearn/utils.py

```diff
@@ -7,20 +7,9 @@
 import time
```

Comment:
This portion of the change integrates the new `MlflowAutologgingQueueingClient` (defined in `mlflow/utils/autologging_utils/client.py`) into the `mlflow.sklearn` autologging integration.